tests/llmcompressor/transformers/obcq/test_mask_structure_preservation.py

```diff
@@ -56,8 +57,9 @@ def test_mask_structure_preserved(self):
         tolerance = 1e-3
         num_calibration_samples = 16
 
+        first_model = AutoModelForCausalLM(self.model)
         oneshot(
-            model=self.model,
+            model=first_model,
```

Comment:
you should still be able to pass in a string

Reply:
ok